### ISSUE TITLE
[CoE Starter Kit - BUG] Orphaned Power Platform objects

### ISSUE THREAD

**Issue by MarcosCostarella** [microsoft/coe-starter-kit#2959]
### Describe the issue

I have been getting the email with orphaned objects (subject Orphaned Power Platform objects without managers)
![image](https://user-images.githubusercontent.com/106347491/172721070-1aec5244-8173-40dd-bab6-ec5c222d2b20.png)
I notice some apps were tagged as orphaned when the author is still with the company, so I am not sure why the CoE is not able to locate the owners on these cases
One live example is XPONLM Rolodex, you can see in the image it is listed as orphaned but, the owner is still with us
![image](https://user-images.githubusercontent.com/106347491/172721159-02375276-e0bf-49a5-b89d-363ca677b41c.png)
![image](https://user-images.githubusercontent.com/106347491/172721238-6d6b776e-3770-4281-8838-2d93da766bf3.png)

### Expected Behavior

These objects not targeted as orphaned

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.38

### What app or flow are you having the issue with?

CLEANUP - Admin | Sync Template v3 (Orphaned Makers

**Comment by MarcosCostarella** [comment]
Hi Marina and Jenefer, thanks for your suggestions. I realized now Steven was granted user access on my previous screen. In any case I am wondering how the owner is missing, I have not been able to remove nor change the author even when the user is blocked. In fact if I check details app section I can see Steven is the owner, any idea what´s going on here?
![image](https://user-images.githubusercontent.com/106347491/173102623-404f19bf-0235-4c4d-a209-44f8452a04ef.png)
Unfortunately I am getting Changing permissions failed, try to set via product UX when chaging to editor or adding new user
![image](https://user-images.githubusercontent.com/106347491/173102981-f0497874-82df-4800-8fff-8f1041022ba4.png)

**Comment by Jenefer-Monroe** [comment]
Does he stil show as user in this product ux? 
![image](https://user-images.githubusercontent.com/89584709/173113107-479274da-25a7-4937-8ae5-51e972061fed.png)

**Comment by MarcosCostarella** [comment]
Yes. If I go to app settings from product UX, I can see the same picture, please see below
![image](https://user-images.githubusercontent.com/106347491/173118332-36ca1c9b-41c4-4617-9e2e-3fe784465562.png)

**Comment by Jenefer-Monroe** [comment]
ok it looks like that spot in the UX showing him as an Owner is just grabbing a name since there is no owner. He is not the owner and you will need to give him edit permissions and then make him an owner as I've described above.

**Comment by MarcosCostarella** [comment]
I am getting this red banner when I hit **change to editor**
![image](https://user-images.githubusercontent.com/106347491/173151245-0fb46013-884e-48c1-a5de-b7cb91fa91ed.png)

**Comment by Jenefer-Monroe** [comment]
there are many reasons people his this, including not having the flows turned on. 
you can go ahead and set in the UX as shown here.
![image](https://user-images.githubusercontent.com/89584709/173155271-17d5ed9a-99f0-47f9-b62b-8fc10ed643f7.png)
